src/components/Form/Form.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/TypeTodo';
+import users from '../../api/users';
+
+interface Props{
+  addTodo: (todo: Todo) => void
+}
+
+export const Form: React.FC<Props> = ({ addTodo }) => {
+  const [touched, setTouched] = useState({
+    input: false,
+    select: false,
+  });
+  const [todoInfo, setTodoInfo] = useState({
+    id: 0,
+    title: '',
+    completed: false,
+    userId: 0,
+  });
+
+  const reset = () => {
+    setTouched({
+      input: false,
+      select: false,
+    });
+    setTodoInfo({
+      id: 0,
+      title: '',
+      completed: false,
+      userId: 0,
+    });
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!todoInfo.title || todoInfo.userId === 0) {
```

Comment:
```suggestion
    if (!todoInfo.title.trim() || todoInfo.userId === 0) {
```
to avoid this case
<img width="216" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/07d6380c-21e8-46ec-8e6c-1dcae95125c2">